charts/argo-controller/Chart.yaml

```diff
@@ -14,6 +14,6 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.2.0
+version: 0.2.1
```

Comment:
I think I saw on another PR you left a comment about a "TODO" that involved automating the version bump to trigger on every merge. I'm curious if it's possible to get something similar but triggered by a merge on another repo (for this use case)?

Reply:
In this case it'd be tougher, because we don't have `sbt` to auto-calculate the versions based on git. It's low enough overhead here that I think we'll be ok manually bumping versions for now. It'd be cool if we could add a CI check that ensures the version is changed whenever a file in the chart is updated.